OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -392,6 +392,23 @@ public func hasHeaderNamed(_ name: String, value: String) -> OHHTTPStubsTestBloc
   }
 #endif
 
+/**
+ * Matcher testing that the `NSURLRequest` body contains a JSON object with the same keys and values
+ * - Parameter jsonObject: the JSON object to expect
+ *
+ * - Returns: a matcher that returns true if the `NSURLRequest`'s body contains a JSON object with the same keys and values as the parameter value
+ */
+#if swift(>=3.0)
+public func hasJsonBody(_ jsonObject: [AnyHashable : Any]) -> OHHTTPStubsTestBlock {
+  return { req in
+    guard let jsonBody = (try? JSONSerialization.jsonObject(with: (req as NSURLRequest).ohhttpStubs_HTTPBody(), options: [])) as? [AnyHashable : Any] else {
```

Comment:
We very recently fix that need for the cast to `NSURLRequest` (see https://github.com/AliSoftware/OHHTTPStubs/pull/260) so `as` can be removed to simplify this expression :)